Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -552,7 +560,7 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
             region: scheme.runAction?.options.region,
             launchAutomaticallySubstyle: launchActionConstants.launchAutomaticallySubstyle,
             storeKitConfigurationFileReference: storeKitConfigurationFileReference,
-            customLLDBInitFile: scheme.runAction?.customLLDBInitFile
+            customLLDBInitFile: customLLDBInitFilePath?.pathString
```

Comment:
from testing locally, the last tweak needed to get this working is prepending `$(SRCROOT)` otherwise it doesn't seem Xcode is picking up the lldbinit file

```suggestion
            customLLDBInitFile: customLLDBInitFilePath.map { "$(SRCROOT)/\($0.pathString)" }
```

Reply:
@kwridan 

thank you, definitely customLLDBInitFile in scheme need `$(SRCROOT)` prefix for relation of path.

Reply:
now `fixture` thing remains. 😄